src/services/googleApi.ts

```diff
@@ -0,0 +1,22 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { REACT_APP_API_BASE_URL } from '../data';
+
+const baseUrl = REACT_APP_API_BASE_URL as string;
```

Comment:
@musllim This will solve the issue with type assertion and it will be easier to debug.
```suggestion
if (!REACT_APP_API_BASE_URL) {
  throw new Error("REACT_APP_API_BASE_URL is not defined");
}

const baseUrl = REACT_APP_API_BASE_URL;
```